src/client/array/dc_array.c

```diff
@@ -871,9 +873,8 @@ dc_array_close(tse_task_t *task)
 	/** Create task to close object */
 	rc = daos_task_create(DAOS_OPC_OBJ_CLOSE, tse_task2sched(task),
 			      0, NULL, &close_task);
-	if (rc != 0) {
-		D_ERROR("Failed to create object_close task "DF_RC"\n",
-			DP_RC(rc));
+	if (rc) {
```

Comment:
Same style for all instancies.
```suggestion
	if (rc != 0) {
```

Reply:
unfortunately we mix those a lot everywhere. we can create a ticket to address this separately.